hash/apple2_flop_orig.xml

```diff
@@ -18588,4 +18588,3051 @@ license:CC0
+		<!-- It requires a Apple IIgs, //c+, or 128K //e with a compatible drive controller card. -->
+		<!--"Science Corner: Space Probe" is a 1989 educational program developed by Peter Andrews, Rob Shaver, Henry F. Olds, Jr., Angela Faeth, Marianne Dunne, and Nancy Davis, and distributed by Society for Visual Education. It requires a Apple IIgs, //c+, or 128K //e with a compatible drive controller card.-->
+
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="1296247">
+				<rom name="science corner- space probe.woz" size="1296247" crc="6bf3b71e" sha1="1d50c90411be9fd18514c6d2ddb6f35334758529"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="grapple">
+		<description>Grapple</description>
+		<year>1982</year>
+		<publisher>Insoft</publisher>
+		<info name="release" value="2022-02-08"/>
+		<sharedfeat name="compatibility" value="A2,A2P,A2E,A2EE,A2C,A2GS"/>
+		<!-- It runs on any Apple II with 48K. -->
+		<!--"Grapple" is a 1982 action game developed by Tony Lewis and distributed by Insoft. It runs on any Apple II with 48K.-->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="354570">
+				<rom name="grapple.woz" size="354570" crc="1e8da238" sha1="554833930d430b51c839018ad533c6dfa00ee3e9"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="aoidchpo">
+		<description>Apple-Oids and Chipout</description>
+		<year>1981</year>
+		<publisher>California Pacific Computer</publisher>
+		<info name="release" value="2022-02-08"/>
+		<sharedfeat name="compatibility" value="A2,A2P,A2E,A2EE,A2C,A2GS"/>
+		<!-- It runs on any Apple II with 48K. -->
+		<!--"Apple-Oids and Chipout" is a 1981 pair of action games developed by Tom Luhrs and distributed by California Pacific Computer. It runs on any Apple II with 48K.-->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="201004">
+				<rom name="apple-oids and chipout.woz" size="201004" crc="e1b224a9" sha1="5bcbdb1329051b2ae2eaf3fa35159a2c9a06d911"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="swdsheol">
+		<description>Sword of Sheol</description>
+		<year>1982</year>
+		<publisher>Winner's Circle</publisher>
+		<info name="release" value="2022-02-08"/>
+		<sharedfeat name="compatibility" value="A2,A2P,A2E,A2EE,A2C,A2GS"/>
+		<!-- It runs on any Apple II with 48K. -->
+		<!--"Sword of Sheol" is a 1982 action game developed by David Riggle and distributed by Winner's Circle. It runs on any Apple II with 48K.&lt;br /&gt;&lt;br /&gt;A 1987 textfile called "Eggs and Keys from Hot Rod" listed "Sword of Sheol" among a list of "vaporwares"; crediting it to Winner's Circle, creators of Sanitron.&lt;br /&gt;&lt;br /&gt;&lt;div&gt;Hot Rod's document is likely based on the advertisement for Sword of Sheol that was included in the August 1983 issue of &lt;i&gt;Creative Computing&lt;/i&gt;, where it appeared next to titles Ice Cave, Exterminator, and Sanitron.&lt;/div&gt;-->
```

Comment:
Is there copy/pasted html in this comment that needs cleaned up?

Reply:
The data entry from the Internet Archive upload is inserted as-is with no additional cleanup.

Reply:
> The data entry from the Internet Archive upload is inserted as-is with no additional cleanup.

Yeah, but that makes it borderline unreadable, with the insanely long lines and HTML entities.